HACKING.md

```diff
@@ -39,3 +39,17 @@ To run the unit tests:
 To run the integration tests:
 
     ./integration-test.sh
+
+### Spark
+- ##### Format namenode
+    Before formatting namenode, datanode must be formatted. To format namenode and datanode:
+
+        docker-compose -f docker/docker-compose.spark.yml -p listenbrainzspark run --rm hadoop-master hdfs namenode -format -clusterId <datanode cluster ID>
```

Comment:
Can we add thiis command to `spark_develop.sh`? Like we have different commands for develop.sh.